dojo-dungeon/src/dojo/rooms/rooms/riddle/Fragen_Pattern.java

```diff
@@ -76,7 +82,7 @@ private Entity questBoss() throws IOException {
     // add boss
     zauberer = new Entity("Zauberer von Patternson");
     zauberer.add(new PositionComponent());
-    zauberer.add(new DrawComponent(new SimpleIPath("character/wizard")));
+    zauberer.add(new DrawComponent(new SimpleIPath("character/monster/orc_shaman")));
```

Comment:
diese änderung ist ok, sollte aber in einen eigenen pr. 

bitte auch den aufgabentext entsprechend anpassen in dem pr.